examples/tutorial/src/vote.ts

```diff
@@ -0,0 +1,20 @@
+import { Wallet } from '@ethersproject/wallet';
+import { VocdoniSDKClient, Vote } from '@vocdoni/sdk';
+
+export async function castVotes (voters: Wallet[], client: VocdoniSDKClient) {
+  for await (const voter of voters) {
+    client.wallet = voter;
+    // Create a vote for option 0 or 1
+    const vote = new Vote([Math.round(Math.random())]);
+    await client.submitVote(vote).then(voteId => {
+      console.log('Vote sent! Vote id: ', voteId);
+      console.log('Verify vote at https://stg.explorer.vote/verify/#/' + voteId);
+    });
+  }
+}
+
+export async function countVotes (client: VocdoniSDKClient) {
+  client.fetchElection().then(election => {
+    console.log('Election results: ' + election.results);
```

Comment:
Beware that `election.results` has the raw ballot protocol results. Maybe for an standard election can be seen easy, but for other type of elections is confusing to understand. The results are also computed in the `metadata` where the questions are, which is way easier to understand. This part is probably not well documented and therefore it's normal to use the `results` field as the results.